lust/lib/widgets/getTextBox.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class TextBox {
+  //TextBox(this._text);
+  //final String _text;
+  //String inputUser;
+
+  static Widget getTextBox(BuildContext context, String _title, String _hint) {
+    String inputUser;
+
+    return new Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: <Widget>[
```

Comment:
mhm. Maybe try:
Dont use `children: <Widget>[`
use only `children: [`

Thats how I did it in the Capacity view